autorest/azure/environments.go

```diff
@@ -126,5 +135,19 @@ func EnvironmentFromName(name string) (Environment, error) {
 	if !ok {
 		return env, fmt.Errorf("autorest/azure: There is no cloud environment matching the name %q", name)
 	}
+
+	if strings.EqualFold(name, "AZURESTACKCLOUD") == true {
+
+		println("autorest/azure: Reading Azure Stack Cloud config from directory", path.Join("etc", "kubernetes", "azurestackcloud.json"))
+		fbytes, err := ioutil.ReadFile(path.Join("/etc", "kubernetes", "azurestackcloud.json"))
```

Comment:
We should be very deliberate about adding references to third party placement of these files. I'd really rather the path where you can find the environment file be a parameter to a constructor.

Reply:
Sure. Changed Kubernetes to azurestackcloud.

Reply:
Sure. Changed kubernetes to azurestackcloud.